qa/suites/rados/mgr/tasks/workunits.yaml

```diff
@@ -13,4 +13,4 @@ tasks:
   - workunit:
       clients:
         client.0:
-          - mgr
\ No newline at end of file
+          - mgr/test_localpool.sh
```

Comment:
explain this change?

Reply:
Earlier, the `test_per_module_finisher.sh` was placed under `mgr` directory where `test_localpool.sh` is present. It's not longer relevant. I will remove this change.

Reply:
k

Reply:
Sorry this is still relevant for the same reason stated above. If not, this test would run both `test_localpool.sh` and `test_per_module_finisher.sh`.